packages/google_sign_in/google_sign_in_platform_interface/lib/src/types.dart

```diff
@@ -49,9 +50,21 @@ class SignInInitParameters {
   /// By default, the list of accounts will not be restricted.
   final String? hostedDomain;
 
-  /// The client ID to use when signing in.
+  /// The OAuth client ID of the app.
+  ///
+  /// The default is null, which means that the client ID will be sourced from a
+  /// configuration file, if required on the current platform. A value specified
+  /// here takes precedence over a value specified in a configuration file.
```

Comment:
nits: Could you add a link to how to set the values with configuration file?  Something like:
```
/// See also: 
/// 
/// [Platform Integration](https://github.com/flutter/plugins/tree/main/packages/google_sign_in/google_sign_in#platform-integration), where you can find the details about the configuration files.
```